TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -0,0 +1,40 @@
+package org.firstinspires.ftc.teamcode.components
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import org.firstinspires.ftc.teamcode.api.TriWheels
+import kotlin.math.PI
+import kotlin.math.atan2
+import kotlin.math.sin
+import kotlin.math.sqrt
+
+object TeleOpMovement : Component {
+    override fun loop(opMode : OpMode) {
+
+        // renaming gamepad1
+        val gamepad = opMode.gamepad1
+        var rotationPower = 0.6 * -gamepad.right_stick_x.toDouble()
+
+        // right bumper and left bumper movement
+        // will slowly turn the robot
+        if (gamepad.left_bumper) {
+            rotationPower += 0.3
+        } else if (gamepad.right_bumper) {
+            rotationPower -= 0.3
+        }
```

Comment:
For the `0.6` and `0.3`s, can we move those to `const val`s to be less hard-coded?

Like:

```kotlin
private const val FINE_ROTATION: Double = 0.3

if (gamepad.left_bumper) {
    rotationPower += FINE_ROTATION
}
```

In the future I want to be able to configure these with FTC Dashboard, but we don't need to do that right now.